.hlint.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 - functions:
-  - {name: unsafePerformIO, within: [PlutusPrelude, Language.PlutusCore.Generators.Internal.Entity, Language.Plutus.CoreToPLC.Plugin]}
-  - {name: error, within: [Main, PlutusPrelude, Language.PlutusCore.StdLib.Meta, Evaluation.Constant.Success, Language.PlutusCore.Constant.Apply, Language.PlutusCore.Constant.Typed, Language.PlutusCore.Evaluation.CkMachine, Language.PlutusCore.TypeSynthesis, Language.PlutusCore.Generators.Internal.Entity, Language.PlutusCore.Generators.Internal.Utils, Language.PlutusCore.Constant.Make, Language.PlutusCore.TH, Language.Plutus.CoreToPLC.Utils, Language.PlutusIR.Compiler.Datatype]}
+  - {name: unsafePerformIO, within: [PlutusPrelude, Language.PlutusCore.Generators.Internal.Entity, DynamicBuiltins.OnEach, Language.Plutus.CoreToPLC.Plugin]}
+  - {name: error, within: [Main, PlutusPrelude, Language.PlutusCore.StdLib.Meta, Evaluation.Constant.Success, Language.PlutusCore.Constant.Apply, Language.PlutusCore.Constant.Typed, Language.PlutusCore.Evaluation.CkMachine, Language.PlutusCore.TypeSynthesis, Language.PlutusCore.Generators.Internal.Entity, Language.PlutusCore.Generators.Internal.Utils, Language.PlutusCore.Constant.Make, DynamicBuiltins.Char, Language.PlutusCore.TH, Language.Plutus.CoreToPLC.Utils, Language.PlutusIR.Compiler.Datatype]}
```

Comment:
We can probably stop banning `error` altogether, no?

Reply:
If anyone agrees, I can remove it. @michaelpj, what do you think?

Reply:
Hm, `error` is still usually a code smell, so I somewhat like having the inconvenience of having to put it in the hlint file.

Reply:
I won't touch it then.